### ISSUE TITLE
Can't start on macOS

### ISSUE THREAD

**Issue by andreyoit** [webchain-network/webchain-wallet#178]
**Describe the bug**
Can't start on macOS, missing Electron Framework

**To Reproduce**
1. Start Wallet app in a Mac that doesn't have Electron Framework

Stacktrace:

> Process:               WebchainWallet [8735]
Path:                  /Applications/WebchainWallet.app/Contents/MacOS/WebchainWallet
Identifier:            com.ethereumclassic.emeraldwallet
Version:               1.2.2 (1.2.2)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           WebchainWallet [8735]
User ID:               501

Date/Time:             2019-12-30 22:13:58.038 +0100
OS Version:            Mac OS X 10.15 (19A536g)
Report Version:        12
Anonymous UUID:        D921CEBD-2A4C-3637-DB7A-979C3D68E4AF

Sleep/Wake UUID:       0E411F91-192B-4D3B-B32A-D6F5DB973948

Time Awake Since Boot: 420000 seconds
Time Since Wake:       3900 seconds

System Integrity Protection: disabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/Electron Framework.framework/Electron Framework
  Referenced from: /Applications/WebchainWallet.app/Contents/MacOS/WebchainWallet
  Reason: no suitable image found.  Did find:
	/Applications/WebchainWallet.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: file too short
	/Applications/WebchainWallet.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: stat() failed with errno=1

**Desktop (please complete the following information):**
 - OS: macOS 10.15

**Comment by danielwebimp** [comment]
Ok thanks for the information we will check it.

**Comment by nativehyun** [comment]
Same problem here. Also Happens on the latest release.

**Comment by danielwebimp** [comment]
Yes, thanks we will check it again. But on Mac 10.13 it works without any issue.
